products/angular-js.md

```diff
@@ -0,0 +1,137 @@
+
+auto:
+  - npm: angular
+
+identifiers:
+  - repology: angular.js
+  - purl: pkg:npm/angular
+
+releases:
+-   releaseCycle: "1.8"
+    releaseDate: 2020-06-04
+    support: 2021-12-31
+    extendedSupport: true
+    latest: "1.9.2"
+    latestReleaseDate: 2023-07-12
+    lts: false
+    discontinued: false
+    link: https://xlts.dev/blog/2023-07-12-xlts-for-angularjs-1-9-2?utm_source=endoflife.date&utm_medium=open-source-pr&utm_campaign=xlts-angularjs-blog&utm_content=pr-link
+
+-   releaseCycle: "1.7"
+    releaseDate: 2018-04-30
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.7.9"
+    latestReleaseDate: 2019-11-19
+    lts: false
+    discontinued: true
+    link: https://code.angularjs.org/1.7.9/docs/misc/version-support-status
+
+-   releaseCycle: "1.6"
+    releaseDate: 2016-12-08
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.6.10"
+    latestReleaseDate: 2018-04-17
+    lts: false
+    discontinued: true
+    link: https://code.angularjs.org/1.6.10/docs/misc/version-support-status
+
+-   releaseCycle: "1.5"
+    releaseDate: 2016-02-17
+    support: 2021-12-31
+    extendedSupport: true
+    latest: "1.5.17"
+    latestReleaseDate: 2023-04-14
+    lts: false
+    discontinued: false
+    link: https://xlts.dev/blog/2023-02-07-xlts-for-angularjs-1-5-x-1-5-17?utm_source=endoflife.date&utm_medium=open-source-pr&utm_campaign=xlts-angularjs-blog&utm_content=pr-link
+
+-   releaseCycle: "1.4"
+    releaseDate: 2015-04-13
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.4.14"
+    latestReleaseDate: 2016-10-11
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.4.14
+
+-   releaseCycle: "1.3"
+    releaseDate: 2014-10-13
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.3.20"
+    latestReleaseDate: 2015-09-29
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.3.20
+
+-   releaseCycle: "1.2"
+    releaseDate: 2013-11-06
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.2.32"
+    latestReleaseDate: 2016-10-11
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.2.32
+
+-   releaseCycle: "1.1"
+    releaseDate: 2012-08-13
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.1.5"
+    latestReleaseDate: 2013-05-22
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.1.5
+
+-   releaseCycle: "1.0"
+    releaseDate: 2010-10-20
+    support: 2021-12-31
+    eol: 2021-12-31
+    extendedSupport: false
+    latest: "1.0.8"
+    latestReleaseDate: 2013-08-22
+    lts: false
+    discontinued: true
+    link: https://github.com/angular/angular.js/releases/tag/v1.0.8
```

Comment:
I do think this is the link that should be there, but there are some problems:
- No change log for V1.1 https://github.com/angular/angular.js/blob/v1.1.x/CHANGELOG.md
- For 1.5, 1.6, 1.7, and 1.8 you can see the changelog on raw mode only because the changelog is too long to render

What are your thoughts on those points?

Reply:
Potencialy solved on [01ce7d6](https://github.com/endoflife-date/endoflife.date/commit/01ce7d69a2d0daacfe0032608f204a396fadbedb)

Reply:
Solved on [05318d3](https://github.com/endoflife-date/endoflife.date/commit/05318d3cd0f0ef0b1c546700811665cb8995cc57)